### PR TITLE
[GDP-279] Fix dates in hmi and minor fix to readme

### PR DESCRIPTION
**Pull request by RobertAJMarshall**
[GDP-279] Update copyright on gdp apps screen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# GENIVI Demo Platform HMI
-This repository holds source code for the GENIVI Demo Platform (GDP) launcher application and HMI. This software is used to demonstrate lower level GENIVI APIs as well as an interface to the GDP itself.
+# GENIVI DeVelopment Platform HMI
```
Comment (tom--pollard) [inline README.md]: Small typo here with the 'V' . Other than that +1

Comment (RobertAJMarshall) [inline README.md]: Oops!

**Comment by tom--pollard** [comment]
If possible I'd like to see this rebased / amended as a single commit (even if a new PR), and if possible the jira tag removed from the commit title (but that's just me being fussy :) )

**Comment by RobertAJMarshall** [comment]
Is that better?

**Comment by tom--pollard** [comment]
Heh, much cleaner, +1 :)

**Comment by tom--pollard** [comment]
Hey @jeremiah seems I don't have write access here, please could you grant it or merge this :)

**Comment by jeremiah** [comment]
I guess this was solved? Sorry I didn't see it sooner -- I was on vacation. ;-)

**Pull request merged**